src/Microsoft.AspNetCore.Localization/project.json

```diff
@@ -1,6 +1,10 @@
 {
   "version": "1.0.0-*",
-  "description": "Middleware for automatically applying culture information to HTTP requests.",
+  "description": "ASP.NET Core middleware for automatically applying culture information to HTTP requests. Culture information can be specified in the HTTP header, query string, cookie, or custom source.",
```

Comment:
Do we really want to get so specific about how culture information can be specified here? Seems like a deeper dive that might more appropriate for docs to me, and like a thing we would forget to update if we added or removed ways to specify localization.